src/plugins/external_tool_support/src/ExternalToolManager.h

```diff
@@ -80,8 +80,9 @@ private slots:
     void innerStart();
     void checkStartupTasksState();
     void markStartupCheckAsFinished();
-    QString addTool(ExternalTool *tool);
+    void addTool(ExternalTool *tool);
     bool dependenciesAreOk(const QString &toolId);
+    QString addToolToListsAndReturnToolPath(const ExternalTool &tool);
```

Comment:
Add comments for every new method created

Reply:
Do we have a codestyle? Where can I find it? Where should the private class method comment be: on the same line or on the line above? Does it have to be in a special format (like Doxygen or your custom, not usual "//")?

Reply:
There is no strict code style. I use:
`
/** Comment text */
`

Reply:
Our codestyle is defined with .clang-format file. If some situation is not defined, feel free to do as you want.